common/changes/@microsoft/rush/ianc-loc-plugin-prototype_2019-05-13-07-53.json

```diff
@@ -1,11 +1,11 @@
 {
```

Comment:
Was this accidentally carried over from the `rush list --json` PR?

Reply:
Oh that's weird. Should go away after Rush is published.